### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by CptanPanic** [sketchpunk/opencomicreader#71]
If this is an open source project, needs to have a license.  https://choosealicense.com/

**Comment by ds17f** [comment]
Yeah, please state the license so we can know if this is usable.  As it stands I'd love to do some work on this project but I have no idea whether I can.
